src/sentry/api/serializers/models/project.py

```diff
@@ -66,6 +68,17 @@
 LATEST_DEPLOYS_KEY = "latestDeploys"
 
 
+def _get_team_memberships(team_list: Sequence[Team], user: User) -> Iterable[int]:
+    """Get memberships the user has in the provided team list"""
+    if not user.is_authenticated:
```

Comment:
when would this case ever happen?

Reply:
That's probably vestigial from when this was a public method. I'll poke around a little and see if we can delete it.